### PR TITLE
gemtopnm: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.nl/common/gemtopnm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# gemtopnm
+
+> Converteer een GEM plaatje naar een PNM plaatje.
```
Comment (leonvsc) [inline pages.nl/common/gemtopnm.md:3]:
```suggestion
> Converteer een GEM afbeelding naar een PNM afbeelding.
```

File: pages.nl/common/gemtopnm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Meer informatie: <https://netpbm.sourceforge.net/doc/gemtopnm.html>.
+
+- Converteer een GEM plaatje naar een PNM plaatje:
```
Comment (leonvsc) [inline pages.nl/common/gemtopnm.md:6]:
```suggestion
- Converteer een GEM afbeelding naar een PNM afbeelding:
```

File: pages.nl/common/gemtopnm.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`gemtopnm {{pad/naar/bestand.img}} > {{pad/naar/uitvoer.pnm}}`
+
+- Beschrijf de inhoud van een gespecificeerd GEM plaatje:
```
Comment (leonvsc) [inline pages.nl/common/gemtopnm.md:10]:
```suggestion
- Beschrijf de inhoud van een gespecificeerde GEM afbeelding:
```

**Review by leonvsc — APPROVED** [review]
LGTM after the suggestions.

**Review by gutjuri — APPROVED** [review]

**Pull request merged**